### ISSUE TITLE
Unable to send custom email - "A message is required."

### ISSUE THREAD

**Issue by matbcvo** [mautic/mautic#14329]
### Mautic Series

5.2.x series

### Mautic installed version

5.2.0

### Way of installing

Other (please specify if relevant in the bug report)

### PHP version

8.3.14

### What browsers are you seeing the problem on?

Chrome

### What happened?

I am unable to send a custom email to a contact. When I try to send it, an error message appears saying, `A message is required.`, even though I have filled in the HTML Code field.

![image](https://github.com/user-attachments/assets/40c218b3-b05f-4515-995c-2c4119b04461)

This Mautic instance has been updated from version 5.1.1 to 5.2.0.

### How can we reproduce this issue?

Step 1. Select a contact.
Step 2. Click the "Send Email" button.
Step 3. Fill in the required fields.
Step 4. Click the "Send" button.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by andersonjeccel** [comment]
Duplicate of https://github.com/mautic/mautic/issues/13987
